Functions/xASL_bids_BIDS2Legacy.m

```diff
@@ -295,7 +297,25 @@
     xASL_Copy(fullfile(pathStudy, 'Participants.tsv'),fullfile(pathLegacy, 'Participants.tsv'));
 end
 
-%% 12. Clean up
+%% 12. Add dataset_description.json
+
+% Check rawdata directory
+if xASL_exist(fullfile(pathStudy, 'rawdata'),'dir')==7
+    % Check dataset_description.json
+    if xASL_exist(fullfile(pathStudy, 'rawdata', 'dataset_description.json'),'file')==2
+        % Load dataset_description.json
+        datasetDescription = spm_jsonread(fullfile(pathStudy, 'rawdata', 'dataset_description.json'));
+        % Add "GeneratedBy" field
+        datasetDescription.GeneratedBy = struct;
+        datasetDescription.GeneratedBy.Name = 'xASL-BIDS';
```

Comment:
Thinking if `xASL-BIDS` is the name we want. It might sound as `conversionASL-BIDS` not making the link to ExploreASL. Plain `ExploreASL` might be better. @HenkMutsaerts what do you think?

Reply:
Sure, changed to ExploreASL in e7ee311 👍